packages/coresandbox/src/components/Slots/SlotTest.tsx

```diff
@@ -0,0 +1,12 @@
+import { Container } from 'semantic-ui-react';
```

Comment:
@sneridagh This is just for test but given the ability to use a container component do we still want to have imports from semantic with new code that we want to merge?

Reply:
I know! :) I was reluctant too. What do we put, then? A custom style?